rollup/plugins/bundlePlugins.js

```diff
@@ -63,6 +63,14 @@ export function makeIsDebugBuildPlugin(includeDebugging) {
   });
 }
 
+export function makeIsCDNBundlePlugin(isCDNBundle) {
+  return replace({
+    values: {
+      __SENTRY_CDN_BUNDLE__: isCDNBundle,
```

Comment:
We could think about making it more explicit that this takes a string but I don't care too much: `isCDNBundle ? 'true' : 'false'`